plugins/thunderbolt/thunderbolt.quirk

```diff
@@ -13,3 +13,8 @@ Flags = retimer-offline-mode
 # Google Rex
 [​​fe682559-28dc-58b3-bf61-aa8414d9f363]
 Flags = retimer-offline-mode
+
+# Google Screebo
+[967000fc-42df-5a17-a0a2-70a43738e78b]
+Flags = retimer-offline-mode
+ThunderboltPort = usb4_port3
```

Comment:
_why_ is this port different?  what happens when you interact with the others?  why can't the kernel export information that we can discover for which port to use for a given update?  should we be embedding the port in the device string instead perhaps?